### DISCUSSION TITLE
How to plot different ObservedProperties in the same graphic

### DISCUSSION THREAD

**Discussion by PedroSolino** [FraunhoferIOSB/FROST-Server#1331] [Q&A]
Hello, I am using FROST to measure some different parameters in the same place. And for analyse these parameters is good to be able to see them in the same graphic. 
For the html page, I am using the same code from the documentation site, but with the address of my local database ( localhost:8080/FROST-Server/v1.1 ). And in that, I can only plot 1 ObserverdProperty in each graphic. 
I know from this example : https://api4inspire.k8s.ilt-dmz.iosb.fraunhofer.de/servlet/is/127/  that is possible to plot different ObservedProperties in the same graphic. But I do not know how to do it. Someone can help me with the html code?

`<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">

    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>

    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
        integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
        crossorigin="" rel="stylesheet" />
    <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
        integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
        crossorigin=""></script>

    <script src="https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/Leaflet.fullscreen.min.js"></script>
    <link href="https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/leaflet.fullscreen.css" rel="stylesheet" />

    <script src="https://unpkg.com/sta-map"></script>

    <style>
        html, body {height:100%;}
        #mymap {
            height: 85%;
            width: 100%;
            background-color: #CCC;
        }
    </style>

    <title>Demo Map</title>
</head>
<body>

<h1>Demo Map</h1>

<div id="mymap"></div>

<script type="text/javascript">
 var mymap = L.map('mymap').setView([49.0, 8.4], 5);
    mymap.addControl(new L.Control.Fullscreen());
    L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
        attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors',
        maxZoom: 18
    }).addTo(mymap);
	L.stam({
        baseUrl: "https://localhost:8080/FROST-Server/v1.1",
        cluster: true,
        clusterMin: 10,
        queryObject: {
            entityType: 'Things'
        }
    }).addTo(mymap);
	
</script>

</body>
</html>
`

**Comment by hylkevds** [comment]
The STA-Map library does not have this functionality out of the box, you'll need to code that yourself. The WebGenesis content management system at the example URL you mentioned doesn't use STA-Map, but a lot of non open source javascript to handle the graphs.
